api/src/main/resources/liquibase.xml

```diff
@@ -94,4 +94,40 @@
     <changeSet author="slubwama" id="patientqueue_1566208965241-17">
         <addForeignKeyConstraint baseColumnNames="creator" baseTableName="patient_queue" constraintName="patient_queue_users_user_id_fk" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="user_id" referencedTableName="users"/>
     </changeSet>
+
+    <changeSet  author="slubwama" id="patientqueue_1566208965241-18">
+        <preConditions onFail="CONTINUE">
```

Comment:
on fail continue or mark run?

Reply:
continue. because you want to keep it open until the condition is executed

Reply:
I do not think this is the intended use of CONTINUE. In your case, mark run seems more appropriate because if the condition fails because of the existence of the table, why would you want to keep/continue running this changeset?

Reply:
@dkayiwa committed the changes